src/main/osd/osd_elements.c

```diff
@@ -945,10 +945,21 @@ static void osdBackgroundHorizonSidebars(osdElementParms_t *element)
 static void osdElementLinkQuality(osdElementParms_t *element)
 {
     uint16_t osdLinkQuality = 0;
-    if (linkQualitySource == LQ_SOURCE_RX_PROTOCOL_CRSF) { // 0-300
-        osdLinkQuality = rxGetLinkQuality()  / 3.41;
-        tfp_sprintf(element->buff, "%c%3d", SYM_LINK_QUALITY, osdLinkQuality);
-    } else { // 0-9
+    if (linkQualitySource == LQ_SOURCE_RX_PROTOCOL_CRSF)
+    { // 0-300
```

Comment:
Also this comment is no longer valid. And yes this should be changed this coding style:
```
if (condition) {
} else {
}
```

Reply:
will do